packages/kotti-ui/source/kotti-inline-edit-2/types.ts

```diff
@@ -0,0 +1,24 @@
+import escape from 'lodash/escape'
+import { z } from 'zod'
+export const DEFAULT_POST_PARSER = (_: any): any => _
```

Comment:
Should contain the actual implementation. IMO setting this directly as a default in `zod` is also fine since the export won’t be needed anymore.

Reply:
Ye, we reached this conclusion after a bit yesterday

won't be exposed as a prop, just implemented internally to translate the `\n` into `<br/>`